.circleci/config.yml

```diff
@@ -278,21 +278,24 @@ jobs:
     parameters:
       target:
         type: string
+      profile:
+        type: string
+        default: release
```

Comment:
Did you want to change this to `quick-release` by default to reduce CI costs for the immediate term?

Reply:
Yup, it's set to quick-release by the workflow. It's on line 489.